config/docker/cards-publication-docker.yml

```diff
@@ -2,7 +2,23 @@
 spring:
   application:
     name: cards-publication
+  deserializer:
+    value:
+      delegate:
+        class: io.confluent.kafka.serializers.KafkaAvroDeserializer
+  # uncomment to enable Kafka
+#  kafka:
+#    consumer:
+#      group-id: opfab-command
 
+schema:
+  registry:
+    url: http://localhost:8081
```

Comment:
Moet de schema registry standaard in de yml actief staan? zit er bijv al een dockerimage bij die een registry draait op 8081?